lib/active_job/workers_relation.rb

```diff
@@ -0,0 +1,61 @@
+# A relation of workers.
+#
+# Relations are enumerable, so you can use +Enumerable+ methods on them.
+# Notice however that using these methods will imply loading all the relation
+# in memory, which could introduce performance concerns.
+#
+# Internally, +ActiveJob+ will always use paginated queries to the underlying
+# queue adapter. The page size can be controlled via +config.active_job.default_page_size+
+# (1000 by default).
+class ActiveJob::WorkersRelation
```

Comment:
`ActiveJob` doesn't concern itself with workers, only with jobs, so we'd want to keep this under the `MissionControl::Jobs` namespace, just like `MissionControl::Jobs::Worker` or `MissionControl::Jobs::Server`.